src/code/code.pcss

```diff
@@ -11,3 +11,12 @@
   border: 1px solid #c8c8c8;
   border-radius: 4px;
 }
+
+pre code.code_ {
```

Comment:
I don't know how `.use(remarkHighlight, { prefix: 'code_' })` used to get to work, but as for the transformation process of `rehypeHighlight`, it takes actually `.use(rehypeHighlight, { prefix: 'code-' })` (which will generate `<code class="code language-js">`) to recognize it correctly or it will generate `<code class="code_ language-js">`.

Reply:
To use existed CSS like `code_var`, it seems we should use `<code class="code_ language-js">`. I suppose we have to change CSS to avoid ugly codes.

Reply:
Let’s add `code-` in this case (you will need to replace and check `code_*` styles)

Reply:
Fixed it.